amr-wind/boundary_conditions/wall_models/MOSD.H

```diff
@@ -0,0 +1,52 @@
+#ifndef MOSD_H
+#define MOSD_H
+
+#include "AMReX_AmrCore.H"
+#include <AMReX.H>
+namespace amr_wind {
+struct MOSD
+{
+    /*
+     * A dynamic wall model that calculates the stress from wave to wind
+     * based on geometric information of the wave.
+     */
+
+    amrex::Real amplitude;
+    amrex::Real wavenumber;
+    amrex::Real omega;
+    amrex::Real time;
+
+    AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE amrex::Real
+    get_dyn_tau(amrex::Real u_dx, amrex::Real v_dx, amrex::Real xc, amrex::Real unit_nor) const
```

Comment:
const all these arguments.

Reply:
should not make const all these variables, since there value will be changing at each time step. Perhaps only for xc and unit_nor since they are fixed in time.

Reply:
That's not really relevant to the function signature. They are const used in the function, therefore they should be const arguments.